res/openvpn-gui-res-it.rc

```diff
@@ -332,127 +347,126 @@ Completa la richiesta di autorizzazione precedente."
+    IDS_ERR_CONN_SCRIPT_FAILED "Errore script di connessione (exitcode=%ld)."
+    IDS_ERR_RUN_CONN_SCRIPT_TIMEOUT "Errore script di connessione. Timeout dopo %d sec."
+    IDS_ERR_CONFIG_EXIST "Esiste già un file di configurazione '%ls'.\n\
+Non puoi avere più file di configurazione con lo stesso nome, \
 a meno che non risiedano in cartelle differenti."
-    IDS_NFO_CONN_TIMEOUT "Connessione all'interfacia di gestione fallita.\n\
-Consulta il file di log (%ls) per maggiori dettagli."
+    IDS_NFO_CONN_TIMEOUT "Fallita connessione all'interfaccia di gestione.\n\
+Per maggiori dettagli consulta il file del registro eventi (%ls)."
     /* main - Resources */
     IDS_ERR_OPEN_DEBUG_FILE "Errore nell'apertura del file di debug (%ls) per l'output."
     IDS_ERR_CREATE_PATH "Errore nella creazione del percorso %ls:\n%ls"
-    IDS_ERR_LOAD_RICHED20 "Non riesco a caricare RICHED20.DLL."
-    IDS_ERR_SHELL_DLL_VERSION "La versione del tuo shell32.dll è troppo vecchia (0x%lx). Hai bisogno almeno della versione 5.0."
+    IDS_ERR_LOAD_RICHED20 "Impossibile caricare RICHED20.DLL."
+    IDS_ERR_SHELL_DLL_VERSION "La versione del file shell32.dll è troppo vecchia (0x%lx).\nÈ necessario una versione 5.0 o successiva."
     IDS_NFO_SERVICE_STARTED "Servizio OpenVPN avviato."
     IDS_NFO_SERVICE_STOPPED "Servizio OpenVPN arrestato."
     IDS_NFO_ACTIVE_CONN_EXIT "Ci sono ancora connessioni attive che verranno chiuse se esci dall'interfaccia di OpenVPN.\
 \n\nSei sicuro di voler uscire?"
-    IDS_NFO_SERVICE_ACTIVE_EXIT "Al momento sei connesso (il servizio di OpenVPN è in funzione). \
+    IDS_NFO_SERVICE_ACTIVE_EXIT "Al momento sei connesso (il servizio di OpenVPN è in funzione).\n\
 Rimarrai connesso anche se esci dall'interfaccia di OpenVPN.\n\n\
-Vuoi continuare e uscire da dall'interfaccia di OpenVPN?"
+Vuoi continuare ed uscire da dall'interfaccia di OpenVPN?"
 
     /* options - Resources */
-    IDS_NFO_USAGE "--help\t\t\t: Mostra questo messaggio.\n\
---connect cnn \t\t: Connetti a ""cnn"" all'avvio. (va inclusa l'estensione)\n\
+    IDS_NFO_USAGE "--help\t\t\t: Visualizza questo messaggio.\n\
+--connect cnn \t\t: All'avvio connettiti a ""cnn"" (va inclusa l'estensione).\n\
 \t\t\t   Esempio: openvpn-gui --connect office.ovpn\n\
 --command cmd [args]\t: Invia un comando a un'istanza aperta dell'interfaccia.\n\
 Comandi supportati:\n\
-    connect cnn            \t: connetti la configurazione ""cnn""\n\
-    disconnect cnn         \t: disconnetti la configurazione ""cnn""\n\
-    reconnect cnn          \t: riconnetti la configurazione ""cnn""\n\
-    disconnect_all         \t: disconnetti tutte le configurazioni connesse\n\
-    exit                 \t\t: termina l'istanza aperta dell'interfaccia (può chiedere conferma)\n\
-    status cnn           \t\t: mostra lo stato della configurazione ""cnn"" se connessa\n\
+    connect cnn            \t: connetti usando la configurazione ""cnn""\n\
```

Comment:
I think this is borderline. There is no concept of connecting/disconnecting using a specific config. Here the word "configurazione" is more like "profilo", therefore you connect or disconnect this profile. The point is that in OpenVPN GUI this concept is always called "configuration".
For this reason I'd definitely drop "usando" (in this line and below) and would either keep the original form or maybe use "profilo" (or another expression if you have something better in mind)

Reply:
if "configuration" it's "configurazione" and if "profile" it's profilo.
I cannot change the string if didn't change in english.